src/components/ReactTags.js

```diff
@@ -323,7 +323,7 @@ class ReactTags extends Component {
     }
 
     // call method to add
-    this.props.handleAddition(tag);
+    this.props.handleAddition(tag, e);
```

Comment:
can you also add a spec for this change ?

Reply:
What spec to add? More comment than what it already has?

Reply:
@melenaos spec to check if the prop `handleAddition` was called with `tag` and `event`. To give you an idea how it will look, you can check here.
You can stub the method `handleAddition` and simulate `keydown` event which will trigger the `handleAddition` prop to be called.
you can check https://github.com/prakhar1989/react-tags/blob/17d912747a600295242209ae5c375f85dc790a52/__tests__/reactTags.test.js#L529
it will be similar to this spec but instead if passing the handleAddition function, just stub it and check if it was called with the expected params.

Reply:
@melenaos any update ?